docker/docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+
+  mysql:
+    image: mysql/mysql-server:5.7
+    container_name: efgs-federation-gateway-mysql
+    ports:
+      - 3306:3306
+    environment:
+      - MYSQL_DATABASE=fg
+      - MYSQL_ROOT_PASSWORD=admin
+      - MYSQL_USER=fg_adm
+      - MYSQL_PASSWORD=admin
+    volumes:
+    - ./db/:/docker-entrypoint-initdb.d/
+    networks:
+      persistence:
+        aliases:
+          - mysql
+
+  backend:
+    build: ../target/docker/.
+    image: efgs-federation-gateway/backend
+    container_name: efgs-federation-gateway-backend
+    ports:
+      - 8080:8080
+    environment:
+      - DB_URL=jdbc:mysql://mysql:3306/fg
```

Comment:
If you use the spring properties e.g. SPRING_DATASOURCE_URL and not DB_URL you can just overwrite the values in the property file and don't need to change it